.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "mkl-dnn"]
 	path = mkl-dnn
-	url = ../mkl-dnn.git
+url=https://github.com/OpenImageDenoise/mkl-dnn.gitOpenImageDenoise/mkl-dnn.git
```

Comment:
You should not changes existent build behavior. Here you break the fork behavior.